### DISCUSSION TITLE
How to hide the function signatures to make candidate shorter

### DISCUSSION THREAD

**Discussion by Eason0210** [joaotavora/eglot#944] [Q&A]
I use` eglot` + `clangd `for `c++-mode`,  but the function signatures will expand the auto complete window too long.
Is it possible to hide this informations, and keeping `Yasnippet` on ?.

<img width="1089" alt="image" src="https://user-images.githubusercontent.com/47546452/166232344-cd794a70-f3a9-4a92-a3a1-04bde9e6514e.png">

Recently, I can get the expected result by shutdown Yasnippet, but I still need the snippet capability provided by `clangd`

<img width="1083" alt="image" src="https://user-images.githubusercontent.com/47546452/166233357-39cb725a-b463-4a88-bf88-863ea1554b8f.png">

Emacs version : 28.0.50
Eglot: latest commit

**Comment by ssl19** [comment]
https://github.com/joaotavora/eglot/blob/d5e335f299216bdf3fe4b0d71fb6e24596a35a6a/eglot.el#L2549-L2556

**Reply by ssl19** [reply]
Change L2551  to `insertText`

**Reply by Eason0210** [reply]
> Change L2551 to `insertText`

Thanks @ssl19 , but it doesn't work. the result is as below picture:

<img width="957" alt="image" src="https://user-images.githubusercontent.com/47546452/168327780-9dd41077-790e-422a-96de-a803b8cb6947.png">

**Reply by ssl19** [reply]
Sorry to hear that :( 
Previously i used this config with jdtls and it worked.
Then I took a look a clangd returned result's, seemed `insertText` of clangd contained snippets, which differed from jdtls.

So this workround could not be used against clangd, I think.

**Reply by ssl19** [reply]
Maybe you can write a parse function for `insertText`, but i don't know if it would increase input latency :D

**Reply by Eason0210** [reply]
> Maybe you can write a parse function for `insertText`, but i don't know if it would increase input latency :D

Thanks for your advice. I will try it.
